internal/handlers/common/update.go

```diff
@@ -977,7 +1006,10 @@ func validateRenameExpression(update *types.Document) error {
 
 	doc, ok := updateExpression.(*types.Document)
 	if !ok {
-		return NewWriteErrorMsg(ErrFailedToParse, "Modifiers operate on fields but we found another type instead")
+		return commonerrors.NewWriteErrorMsg(
```

Comment:
I guess same about codecov here and one more below: it's a refactoring, so we don't add new tests for it.

Reply:
Wõ can test this one. It's just a simple one like `bson.D{{"$inc", bson.A{}}}`.